src/commcare_cloud/commands/deploy/command.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import
```

Comment:
mostly just moved code from `deploy.py` after extracting out the commcare deploy bits into `commcare.py`